tsconfig.json

```diff
@@ -12,9 +12,9 @@
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
     "outDir": "./lib",
-    "allowJs": true,
     "experimentalDecorators": true,
-    "noErrorTruncation": true
+    "noErrorTruncation": true,
+    "declaration": true
```

Comment:
This should have been enabled previously. Now that it is on, we will see the category of errors this inference pattern raised in google3.